config/api_config.go

```diff
@@ -78,3 +78,8 @@ func (cfg *APIConfig) Load(v *viper.Viper) error {
 	cfg.readIn(v)
 	return v.Unmarshal(&cfg)
 }
+
+// APIEndpoint returns the full URL for a given endpoint
+func APIEndpoint(endpoint string) string {
```

Comment:
Oh, wait. You mean the err from the NewAPIConfig()? I think that looks fine. It really shouldn't error out under normal circumstances (I guess that goes for most things, but still. Some errors are fairly expected, whereas this is going to be a very rare occurrence).